src/main/java/io/codelex/classesandobjects/practice/Movie.java

```diff
@@ -0,0 +1,44 @@
+package io.codelex.classesandobjects.practice;
+
+public class Movie {
+    private String title;
+
+    private String studio;
+    private String rating;
+
+    public static void main(String[] args) {
```

Comment:
Tas nav pareizi, ka klasē Movie no kuras veido objektus liec iekšā main metodi - to vajadzētu atsevišķā klasē - kaut kādā MovieApp vai tamlīdzīg.